{{cookiecutter.project_repo_name}}/Dockerfile

```diff
@@ -16,11 +16,11 @@ COPY . /opt/wps
 
 WORKDIR /opt/wps
 
-# Create conda environment
-RUN conda env create -n wps -f environment.yml
+# Create conda environment with PyWPS
+RUN conda create -n wps -c conda-forge python=3.7 pywps=4.2
```

Comment:
Opps this one still pin pywps to 4.2, not allowing 4.2.4.  But I think we can revert to the previous line now that we properly specify version range in `environment.yml`?

Reply:
I can add the version range here as well ... I thought it would always be pick the latest 4.2.x.

I would not use the existing `environment.yml` ... it has too many dependencies which are not used in the docker container ... make the image larger ...

Reply:
I habe opened a  new PR #93. Please edit or comment there :)